tests/console/regproxy.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Setup regproxy and redirect registry.opensuse.org to it
+# Maintainer: Fabian Vogt <fvogt@suse.com>
+
+use base 'consoletest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+use suse_container_urls 'get_suse_container_urls';
+use version_utils qw(is_sle is_opensuse is_tumbleweed is_leap);
+
+sub run {
+    select_console 'root-console';
+
+    # TODO: Use suse_container_urls
```

Comment:
Please delete again, keep in local WIP commit or ticket

Reply:
Of course, that's why this is a draft PR

Reply:
Sorry, didn't realize, looked like nearly perfect already, :)

Reply:
It's implemented now